imu_complementary_filter/CMakeLists.txt

```diff
@@ -28,6 +28,9 @@ add_library(complementary_filter
 )
 #target_link_libraries(complementary_filter rclcpp sensor_msgs)
 
+set_target_properties(complementary_filter PROPERTIES
+  POSITION_INDEPENDENT_CODE ON
+)
```

Comment:
Yes, this was missing. Alternatively we can build a shared library instead (which already has position independent code by default).

Reply:
Your PR was perfect: It only changed the minimal amount of code to fix the bug. It just triggered me to do some more changes as well. :)